OpenRA.Game/GameRules/Ruleset.cs

```diff
@@ -180,8 +187,12 @@ public static Ruleset LoadDefaultsForTileSet(ModData modData, string tileSet)
 				var sequences = mapSequences == null ? modData.DefaultSequences[tileSet] :
 					new SequenceProvider(fileSystem, modData, ts, mapSequences);
 
-				// TODO: Add support for custom voxel sequences
-				ruleset = new Ruleset(actors, weapons, voices, notifications, music, ts, sequences);
+				var modelSequences = dr.ModelSequences;
+				if (mapModelSequences != null)
+					modelSequences = MergeOrDefault("ModelSequences", fileSystem, m.ModelSequences, mapModelSequences, dr.ModelSequences,
```

Comment:
I'd like to keep it separate to make it clear that it is replacing `modelSequences`, not modifying it.  This is really just a terniary statement, but the MergeOrDefault branch is too ugly to write it as one.